### PR TITLE
Sorting Frame: TBF

### PR DESCRIPTION
**Pull request by Quinn-Barber**
Flush out final changes (Math notation, explanations, syntax highlighting)
Test out sorting code

### CODE REVIEW

File: src/intro-to-c/sorting/sorting.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+have been discovered and progressively improved up to the point where we are at now! In this lesson, we will go over some of the most common sorting algorithms, as well as
+their usefulness to programming!
+
+# Time Complexity
+
+Time complexity is an important concept to understand for sorting algorithms! Each sorting algorithm has its own time complexity which can help identify distinct differences within
+the algorithms themselves.
+
+Let us say we have a certain number of operations throughout a singular loop in our code, such as addition, multiplication, conditional statements, printing, etc.
+If this loop goes through these operations _*n*_ times, we can call this order _*n*_ time complexity, or O(n). This is called big-O notation, and is a useful way of classifying the
```
Comment (jerrettl) [inline src/intro-to-c/sorting/sorting.md]: I would use math mode notation here and onward.

```suggestion
If this loop goes through these operations $n$ times, we can call this order $n$ time complexity, or $O(n)$. This is called big-O notation, and is a useful way of classifying the
```

File: src/intro-to-c/sorting/sorting.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+speed of our code. An example of this is below:
+
+```
```
Comment (jerrettl) [inline src/intro-to-c/sorting/sorting.md]: Syntax highlighting

```suggestion
``` c
```

File: src/intro-to-c/sorting/sorting.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+- Selection Sort \*
+  Time Complexity: O(n^2)
+- Insertion Sort \*
+  Time Complexity: O(n^2)
+- Merge Sort
+  Time Complexity: O(nlog(n))
+- Heap Sort
+  Time Complexity: O(nlog(n))
+- Quick Sort
+  Time Complexity: O(nlog(n))
```
Comment (jerrettl) [inline src/intro-to-c/sorting/sorting.md]: Each bullet will render strangely on one line. You may want to convert it to HTML to see how this comes out and adjust the spacing according to how you want it to look. Maybe make it into a table? (Note to self: add tables to the sample document 😉)

File: src/intro-to-c/sorting/sorting.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+
+The faster ways to sort, Merge, Heap, and Quick Sort are very useful and quick! However, we will not be going over these sorting methods in this section and you are encouraged to go over these in your own time to learn what makes them so fast and effective.
+
+# **Bubble Sort**
```
Comment (jerrettl) [inline src/intro-to-c/sorting/sorting.md]: This is already bold by being a header, and will instead bolden these entries in the table of contents at the top of the page. This is my personal preference, but I would un-bold this text and turn it into a second-level heading so it falls under the "Sorting Algorithms" header.

**Review by jerrettl — COMMENTED** [review]
As I wrote on the Discord, I think this is super awesome. I only have mainly formatting suggestions for you.

You may also want to consider moving this into the `cs1` folder, as this also dives into time complexity.

Great work!

File: src/intro-to-c/sorting/sorting.md (hunk 4 above)
Comment (Quinn-Barber) [reply src/intro-to-c/sorting/sorting.md]: I will test these different options out, thanks!

**Comment by Quinn-Barber** [comment]
Check out the final flushed changes @jerrettl

**Pull request merged**